randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+        ),
+    ],
+    "2271263048": [
+        # Hive Mecha Bridge Tip
+        # Replaces Hive Mecha bridge scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        "&main-color=#89D6FF;To cross the bridge without fighting Hive Mecha, carefully walk along its right edge.",
+    ],
+    "2053272936": [
+        # Boost Pipe Speed Tip
+        # Replaces Tallon Canyon halfpipe scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;When using the &main-color=#FF3333;Boost Ball&main-color=#89D6FF; in a half pipe, "
+            "hold only &image=8B0C22A7; or &image=778CCD6A; and boost at the lowest point to gain the most height."
+        ),
+    ],
+    "399696251": [
+        # Boost Speed Limit Tip
+        # Replaces Ruined Gallery morph tunnel scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;When using the &main-color=#FF3333;Boost Ball&main-color=#89D6FF;, "
+            "if the control stick is kept in a neutral position, the Boost Ball has no speed limit."
+        ),
+    ],
+    "4175379874": [
+        # Flaahgra Wavebuster Tip
+        # Replaced Flaahgra scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "Flaahgra",
+        (
+            "&main-color=#89D6FF;Flaahgra can be defeated with the &main-color=#FF3333;Wavebuster&main-color=#89D6FF;. "
+            "On Normal difficulty, doing so without any other sources of damage requires 230 missiles."
+        ),
+    ],
+    "889553182": [
+        # Frozen Metroid Hop Tip
+        # Replaces Metroid scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "Metroid",
+        (
+            "&main-color=#89D6FF;Jumping on the center of a frozen Metroid causes one to bounce on top of it.\n\n"
+            "When it breaks out of the ice, anyone on top is launched high in the air\n\nThis can be used to traverse "
+            "Metroid Quarantine A without Space Jump Boots, Morph Ball Bombs, or Spider Ball."
+        ),
+    ],
+    "4176439895": [
+        # Frozen Hunter Metroid Hop Tip
+        # Replaces Hunter Metroid scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "Hunter Metroid",
+        (
+            "&main-color=#89D6FF;Jumping on the center of a frozen Hunter Metroid causes one to bounce on top of it."
+            "\n\nWhen it breaks out of the ice, anyone on top is launched high in the air.\n\n"
+            "This can be used to acquire Frost Cave's item without the Grapple Beam."
+        ),
+    ],
+    "425486343": [
+        # Arboretum door DBJ tip
+        # Replaces Arboretum big branch scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;It is possible to surmount the gate "
+            "at the top of this room with a carefully-timed double bomb jump."
```

Comment:
It would be worth noting that the bomb jump must be directed at the top right of the gate.